pkg/odo/cli/pipelines/service/add.go

```diff
@@ -63,6 +63,11 @@ func (o *AddOptions) Run() error {
 		WebhookSecret:            o.webhookSecret,
 	}, ioutils.NewFilesystem())
 
+	if err == nil {
+		log.Successf(fmt.Sprintf("Service %s has been created sucessfully at environment %s.", o.serviceName, o.envName))
```

Comment:
If you look at the documentation for [Successf](https://godoc.org/github.com/openshift/odo/pkg/log#Successf)

This can be simplified to:

```suggestion
		log.Successf("Service %s has been created sucessfully at environment %s.", o.serviceName, o.envName)
```

Reply:
But, I'd probably follow the guidance in https://golang.org/doc/effective_go.html#control-structures and change the code to

```go
if err != nil {
    return err
}
log.Successf("Service %s has been created sucessfully at environment %s.", o.serviceName, o.envName)
return nil

Reply:
Done